playbooks/vars/sample-variables-sap-swpm-default-mode-s4hana-distributed-aas-install.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+# Detect SAP NetWeaver software
+sap_install_media_detect_source_directory: "/software"
+sap_install_media_detect_source: local_dir
+sap_install_media_detect_swpm: true
+sap_install_media_detect_hostagent: true
+sap_install_media_detect_igs: true
+sap_install_media_detect_kernel: true
+sap_install_media_detect_webdisp: false
+
+# Product ID for New Installation
+sap_swpm_product_catalog_id: NW_DI:S4HANA2023.CORE.HDB.PD
+#NW_DI:S4HANA2023.CORE.HDB.PD   - Distributed AAS
+#NW_DI:S4HANA2023.CORE.HDB.ABAPHA   - High Availbility AAS
+
+# Generate inifile.params using sap_swpm Ansible Role
+sap_swpm_inifile_list:
+  - swpm_installation_media
+  - credentials
```

Comment:
Missing `credentials_hana`

Reply:
Hi Sean,
Looking at the jinja template credentials_hana will add HDB_Schema_Check_Dialogs.schemaPassword and
storageBasedCopy.hdb.systemPassword. When comparing to a SWPM-created param file these two are not present. We have successfully installed AAS using the settings above (i.e. without credentials_hana), therefore I don't thing it is required.

Reply:
Fine by me :)